os/include/tinyara/binfmt/binfmt.h

```diff
@@ -371,20 +373,18 @@ int binfmt_exit(FAR struct binary_s *bin);
  *              directory. Otherwise, path must be the absolute path to the
  *              program.
  *
- *   binsize  - The size of ELF binary to be loaded, if this value is less than
- *              will return with invalid parameter.
- *
- *   offset   - The offset from which ELF binary has to be read in MTD partition.
- *
- *   ram_size - The size of RAM partition required to run this app
+ *   load_attr - This structure contains the information that the binary
+ *               needs to load.
  *
  * Returned Value:
  *   This is an end-user function, so it follows the normal convention:
  *   It returns the PID of the exec'ed module.  On failure, it returns
  *   -1 (ERROR) and sets errno appropriately.
  *
  ****************************************************************************/
-int load_binary(FAR const char *filename, size_t binsize, size_t offset, size_t ram_size, size_t stack_size, uint8_t priority);
+#ifdef CONFIG_BINARY_MANAGER
+int load_binary(FAR const char *filename, FAR struct load_attr_s *load_attr);
```

Comment:
You defined `load_attr_t` type and used in binary_manager.h 
How about unifying the type?

Reply:
ok. I modified it.